viewing_party/party.py

```diff
@@ -0,0 +1,196 @@
+def create_movie(movie_title, genre, rating):
+
+    if movie_title and genre and rating:
+        return {"title" : movie_title, "genre" : genre, "rating" : rating}
+    return None
+   
+
+def add_to_watched(user_data, movie):
+  
+    user_data["watched"].append(movie)
+    return user_data
+   
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(janes_data, Title_A):
+
+    for each in janes_data["watchlist"]:
```

Comment:
`each` doesn't give us information about what each item in the list is. A more meaningful name like `movie` would enhance readability.